frigate/test/test_ffmpeg_presets.py

```diff
@@ -156,6 +291,142 @@ def test_ffmpeg_output_record_not_preset(self):
             " ".join(frigate_config.cameras["back"].ffmpeg_cmds[0]["cmd"])
         )
 
+    def test_ffmpeg_output_record_rotate_90_preset(self):
```

Comment:
Have these all been tested? (If not, which ones?) 

At least some of these I believe wouldn't actually work in ffmpeg because they're missing the hwupload/hwdownload formatting args. 

I've spent a lot of time getting these to manually work with rotation with my nvidia and vaapi setups so may need to have the ffmpeg commands for the test confirmed working.